clients/fossid-webapp/src/main/kotlin/model/result/MatchType.kt

```diff
@@ -21,6 +21,7 @@ package org.ossreviewtoolkit.clients.fossid.model.result
 
 enum class MatchType {
     NONE,
+    FAILED,
```

Comment:
Just wondering, is there any ordering in place here? If not, how about adding a commit to sort these, e.g. simply alphabetically?

Reply:
> FossID: Sort the match type enum

For completeness, this should say "FossID: Sort the match type enum *alphabetcially*".

I haven't checked, but are there other enums in the FOSSID code that should get sorted in the same run?

Reply:
jep, I changed them all